tests/manage/monitoring/prometheus/test_alerting_works.py

```diff
@@ -21,6 +25,24 @@ def test_alerting_works():
     assert len(alerts) > 0
 
 
+@pytest.mark.polarion_id("OCS-2503")
```

Comment:
Does this PR cover https://bugzilla.redhat.com/show_bug.cgi?id=1897674? If yes, can you please add bugzilla marker as well.